src/Chapter2.hs

```diff
@@ -621,8 +628,9 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
-
+takeEven :: [a] -> [a]
+takeEven l = [l !! x | x <- [0..(length l) - 1], even x]
```

Comment:
Interesting idea 💡 🎨 
However, it won't be efficient.
Alternatively, you could use pattern matching in `takeEven` instead. As you need to take every second element, it is possible to represent with a few pattern matching clauses 🙂

Reply:
Thank you. I'll take another whack at that one. I really like Haskell's list comprehensions. :sweat_smile: